Tests/Microsoft.AppCenter.Test.Windows/AppCenterTest.cs

```diff
@@ -100,6 +100,30 @@ public void VerifySdkVersion()
             Assert.AreEqual(WrapperSdk.Version, AppCenter.SdkVersion);
         }
 
+        /// <summary>
+        /// Verify country code setter
+        /// </summary>
+        [TestMethod]
+        public void SetCountryCode()
+        {
+            string valideCountryCode = "US";
+            string invalidCountryCcode = "US1";
```

Comment:
2 Typos, `valideCountryCode` should be `validCountryCode` and `invalidCountryCcode` should be `invalidCountryCode` throughout this test.